src/engine/playback.cpp

```diff
@@ -915,7 +915,7 @@ void DivEngine::processRow(int i, bool afterDelay) {
         //printf("\x1b[1;36m%d: extern command %d\x1b[m\n",i,effectVal);
         extValue=effectVal;
         extValuePresent=true;
-        dispatchCmd(DivCommand(DIV_CMD_EXTERNAL,effectVal));
+        dispatchCmd(DivCommand(DIV_CMD_EXTERNAL,i,effectVal));
```

Comment:
How did this even work when I tested???  I guess lucky coincidence.

Reply:
I don't know.